docusaurus/README.md

```diff
@@ -0,0 +1,36 @@
+# Documentation
+
+The documentation is built using [Docusaurus 2](https://docusaurus.io/). The deployment is done through a centralized build from [IOTA WIKI](https://github.com/iota-community/iota-wiki). To run a local instance the [IOTA WIKI CLI](https://github.com/iota-community/iota-wiki-cli) is used.
+
+## Prerequisites
+
+- [Node.js v14.14+](https://nodejs.org/en/)
+- [yarn](https://yarnpkg.com/getting-started/install)
+
+## Installation
+
+```console
+yarn
+```
+
+This command installs all necessary dependencies.
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
```

Comment:
getting error

```
Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)
$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...
```

Reply:
Ahh, that's probably because you are not in the docusaurus folder?

Reply:
Hmm no, that would actually be a different error 🤔

Reply:
@jlvandenhout can you help here?

Also @apenzk you can also always use `Gitpod` to test this stuff, it gives you an environment where we are basically sure it will work 😅 
Just append https://gitpod.io/# to the repo url. In this case just open https://gitpod.io/#https://github.com/Dr-Electron/IOTA-2.0-Research-Specifications/tree/docs/new-build-system

Reply:
Just to be sure: you did run `yarn` and then `yarn start`?

Reply:
yes

Reply:
my error. i ran yarn in the root and then yarn start in /docusaurus. unfortunatelly yarn finishes with 
```
69c9adf812c8ab80b0c42: The remote archive doesn't match the expected checksum
➤ YN0000: └ Completed in 7m 28s
➤ YN0000: Failed with errors in 7m 28s
```
while on gitpod it finishes correctly

Reply:
Your above error seems to be related to the yarn cache. 
You could try something like:
```
yarn cache clean --all
yarn install
```

Reply:
Gitpod is a really nice tool to test everything in a reproducible setup without having to clone everything.
I can add a gitpod config and mention it in the README if you want. In another PR tho ;)